base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize
+    super()
+    @nameable = Nameable.new
+  end
```

Comment:
- [ ] You would need to pass a nameable object as a parameter in the constructor
- [ ] You are required to assign the nameable object to an instance variable. Hints:

```ruby
        class Decorator < Nameable
           attr_accessor :nameable
  
            def initialize(nameable)
               @nameable = nameable
            end
          '
          '
          '
```